cmds/console/port_linux.go

```diff
@@ -0,0 +1,12 @@
+// Copyright 2012-2017 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// This file contains common variables for port IO devices for Linux
+package main
+
+import (
+	"os"
+)
+
+var f *os.File
```

Comment:
Consider renaming this to something more descriptive. I mean, we all know that it's a file, but given that it has package scope I'd expect it to be a local since it's a short name. When I looked at the code for `open()` I was confused as to why `f` was seemingly discarded; it took me a moment to figure out what was going on.